app/src/main/java/com/cornellappdev/android/eatery/onboarding/OnboardingActivity.java

```diff
@@ -25,6 +25,20 @@ protected void onCreate(Bundle savedInstanceState) {
         mViewPager = findViewById(R.id.onboarding_viewpager);
         mViewPager.setAdapter(new OnboardingPagerAdapter(getSupportFragmentManager()));
         mViewPager.setCurrentItem(0);
+        mViewPager.setOnPageChangeListener(new ViewPager.OnPageChangeListener() {
+            @Override
+            public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {
+                // Must be overridden.
```

Comment:
is this a `TODO` or an intentional empty method?

Reply:
It's intentionally empty bc we don't have to do anything with it, but it must be overridden to compile.